components/Landing.tsx

```diff
@@ -1,17 +1,63 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import Link from "next/link";
+import { useTranslations } from "next-intl";
+import { LandingProps } from "@/types/Landing-types";
+
+export default function Landing({ onStart }: LandingProps) {
+  const t = useTranslations("Landing");
+
+  const [selectedLang, setSelectedLang] = useState("");
```

Comment:
It seems we could set the default state to "English"?

Reply:
Not 100% sure about it

Reply:
I'll set the default state to zh-TW as it's the default locale. From my pov, whether setting the default state or not doesn't impact the functionality of the component, but indeed setting the default state can prevent the flicker of the switch button's appearance in the initial rendering if useEffect delayed.